src/main/java/com/jfrog/ide/common/utils/Utils.java

```diff
@@ -41,7 +41,7 @@ public static License toLicense(com.jfrog.xray.client.services.summary.License o
     }
 
     public static Issue toIssue(com.jfrog.xray.client.services.summary.Issue other) {
-        Severity severity = Severity.fromString(other.getSeverity());
+        Severity severity = Severity.FromString(other.getSeverity());
```

Comment:
Any idea how ```fromString``` became ```FromString``` in the build-info code base? :) 
It's a method, not a class. Let's fix.

Reply:
It accidently caused by this [commit](https://github.com/jfrog/build-info/commit/fa23cac834a4c12462431f7b177a593986b587b7#diff-427cd6de62dc2aadcc8b7a91fa7adb1ad877517bffd134b6820be815cc4d062aR50) and fixed by this [PR](https://github.com/jfrog/build-info/pull/440).
I'll wait for the next build-info release to keep it `fromString`, before merging this PR.